crates/node/src/interactive_cli/context.rs

```diff
@@ -79,19 +79,19 @@ enum Commands {
     /// Leave a context
     Leave {
         /// The context to leave (omit to use default context)
-        #[clap(long = "context", short = 'c')]
-        context: Option<Alias<ContextId>>,
+        #[clap(long, short, default_value = "default")]
+        context: Alias<ContextId>,
```

Comment:
`leave` is a destructive action, should not be implicit

 ```suggestion
     Leave {
        /// The context to leave (omit to use default context)
        #[clap(long, short)]
        context: Alias<ContextId>,
```